docs/modules/writing_a_module.md

```diff
@@ -64,4 +64,7 @@ callbacks for.
 Modules can register callbacks using one of the module API's `register_[...]_callbacks`
 methods. The callback functions are passed to these methods as keyword arguments, with
 the callback name as the argument name and the function as its value. This is demonstrated
-in the example below. A `register_[...]_callbacks` method exists for each category.
\ No newline at end of file
+in the example below. A `register_[...]_callbacks` method exists for each category.
+
+Callbacks for each category can be found by clicking the relevant links in the navigation
+panel on the left-hand side of this documentation.
```

Comment:
This sentence might not make sense to people who happen to be viewing the documentation on GitHub or in their terminal, annoyingly.

Additionally, the left-hand sidebar can be closed, which may confuse people as well.

Perhaps this could be reworded as:

> Callbacks for each category can be found on their respective page of the [Synapse documentation website](https://matrix-org.github.io/synapse).